rebar.config

```diff
@@ -3,7 +3,7 @@
   warnings_as_errors,
   warn_untyped_record]}.
 
-{deps, []}.
+{deps, [wts]}.
```

Comment:
this should be specified as a range (i.e. "~>0.1.0") to prevent conflicts with other hex packages.

Reply:
The rebar3 hex plugin will do that automatically when publishing, but I'm fine adding it to be explicit.

Reply:
ah didn't know the hex plugin does that automatically now :), while I personally prefer explicit over implicit so down the road, we can be clear about switching to 0.2.0 (or 1.0.0) it's not something that will break right now :)